CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ All notable changes to this project will be documented in this file. The format
 
 ## Unreleased
 
+## [1.3.0][1.3.0] - 2022-01-17
```

Comment:
Added the date of when your going to release

Reply:
i will update the date when i release it.